meta-genivi-dev/recipes-sota/genivi-swm/genivi-swm_git.bb

```diff
@@ -0,0 +1,67 @@
+DESCRIPTION = "Genivi Software Loading Manager"
+SECTION = "base"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = \
+"file://${S}/LICENSE;md5=8d2127b230519ce4ad2c16070bdeb7b3"
+
+SRCREV = "${AUTOREV}"
+SRC_URI = "\
+  git://github.com/advancedtelematic/genivi_swm;branch=gdp-build;protocol=https \
+  file://package_manager.service \
+  file://partition_manager.service \
+  file://lifecycle_manager.service \
+  file://software_loading_manager.service \
+  file://module_loader_ecu1.service \
+  file://org.genivi.SoftwareLoadingManager.conf \
+  "
+FILESEXTRAPATHS_prepend := "${THISDIR}:"
```

Comment:
Please remove this line and you need to move your service and conf files under "genivi-swm/genivi-swm or genivi-swm/files" directory.